IoTHomeAppSample/IoTHomeApp/Package.appxmanifest

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" IgnorableNamespaces="uap mp">
+  <Identity Name="ShellHomeApp" Publisher="CN=MSFT" Version="1.0.0.0" />
+  <mp:PhoneIdentity PhoneProductId="6db05a29-0188-44f7-bac0-cba49784c9d3" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
+  <Properties>
```

Comment:
you might want to demonstrate how to get back to the IoTHomeApp from App1 and App2 ... could be accomplished by adding a protocol hander here that the other apps call.